app/src/components/TreeView.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+  <div class="tree-container">
+    <h3 :id="id-title" class="tree-title" v-if="title">{{ title }}</h3>
+    <ul :id="id" role="tree" class="tree-hierarchy tree-root" :aria-labelledby="id-title"></ul>
+  </div>
+</template>
+
+<script>
+import * as d3 from 'd3'
+

```

Comment:
It would be nice to explicitly state which parts of d3 are imported. Not sure, but it might also help reduce the size of the bundle through tree shaking.